colin/core/ruleset/ruleset.py

```diff
@@ -162,47 +162,47 @@ def get_checks_path():
     return os.path.abspath(os.path.join(__file__, rel_path))
 
 
-def get_config_file(config=None):
+def get_ruleset_file(ruleset=None):
     """
-    Get the config file from name
+    Get the ruleset file from name
 
-    :param config: str
+    :param ruleset: str
     :return: str
     """
-    config = config or "default"
+    ruleset = ruleset or "default"
 
-    config_directory = get_config_directory()
-    config_file = os.path.join(config_directory, config + JSON)
+    ruleset_directory = get_ruleset_directory()
+    ruleset_file = os.path.join(ruleset_directory, ruleset + JSON)
 
-    if os.path.exists(config_file) and os.path.isfile(config_file):
-        logger.debug("Configuration file '{}' found.".format(config_file))
-        return config_file
+    if os.path.exists(ruleset_file) and os.path.isfile(ruleset_file):
```

Comment:
Isn't `os.path.exists(ruleset_file)` superset of `os.path.isfile(ruleset_file)` ?
I mean, I think just
```python
if os.path.isfile(ruleset_file):
```
should do the same.

Reply:
It is not something related to the pull request, but yes, you're right.